antenna/app.py

```diff
@@ -74,29 +74,26 @@ def filter(self, record):
                 "filters": ["add_hostid"],
             },
         },
-    }
-    if app_config("local_dev_env"):
-        # In a local development environment, we log to the console in a
-        # human-readable fashion
-        dc["loggers"] = {
-            "antenna": {
-                "propagate": False,
-                "handlers": ["console"],
-                "level": app_config("logging_level"),
-            },
-            "markus": {"propagate": False, "handlers": ["console"], "level": "INFO"},
-        }
-        dc["root"] = {"handlers": ["console"], "level": "WARNING"}
-    else:
-        # In a server environment, we log everything in mozlog format
-        dc["loggers"] = {
+        "loggers": {
             "antenna": {
                 "propagate": False,
                 "handlers": ["mozlog"],
                 "level": app_config("logging_level"),
             }
+        },
+        "root": {"handlers": ["mozlog"], "level": "WARNING"},
+    }
+
+    if app_config("local_dev_env"):
+        # In a local development environment, we log to the console in a human-readable
+        # fashion and add a markus logger
+        dc["loggers"]["antenna"]["handlers"] = ["console"]
+        dc["loggers"]["markus"] = {
+            "propagate": False,
+            "handlers": ["console"],
+            "level": "INFO",
         }
-        dc["root"] = {"handlers": ["mozlog"], "level": "WARNING"}
+        dc["root"]["handlers"] = ["console"]
```

Comment:
This removes some redundancy in logging configuration, but doesn't change behavior.